src/app/dashboard/users/page.tsx

```diff
@@ -44,7 +72,19 @@ async function getUsers(
     return
   }
 
-  const settings = getSettings()
+  const session = await getServerSession(authOptions)
+  const userId = session?.user.id
+
+  if (settings.general.isOutsideAccess && !userId) {
+    return users.slice(0, numberOfUsers)
```

Comment:
This stops the additional data further down from being processed. Anonymous and private view should be the same besides the missing highlighted user in the former.

<img width="605" alt="image" src="https://github.com/user-attachments/assets/66ddb81f-0e8b-4c17-bad8-98c092f02901">

<img width="569" alt="image" src="https://github.com/user-attachments/assets/bad73a99-6214-4868-95a3-a25932cae6ba">